resources/assets/js/store/Application/applicationActions.ts

```diff
@@ -237,6 +245,40 @@ export const sendReferenceEmail = (
     { applicationId, referenceType },
   );
 
+export const UPDATE_APPLICATION_STEP_STARTED =
+  "APPLICATION: UPDATE STEP STARTED";
+export const UPDATE_APPLICATION_STEP_SUCCEEDED =
+  "APPLICATION: UPDATE STEP SUCCEEDED";
+export const UPDATE_APPLICATION_STEP_FAILED = "APPLICATION: UPDATE STEP FAILED";
+
+export type UpdateApplicationStepAction = AsyncFsaActions<
+  typeof UPDATE_APPLICATION_STEP_STARTED,
+  typeof UPDATE_APPLICATION_STEP_SUCCEEDED,
+  typeof UPDATE_APPLICATION_STEP_FAILED,
+  { [step in ApplicationStep]: ProgressBarStatus },
+  { applicationId: number; stepId: number }
+>;
+
+export const updateApplicationStep = (
```

Comment:
Can we rename this 'touchApplicationStep', or add a comment explaining that it just sets 'touched' to true, regardless of the Body of the PUT request?